web_responsive/static/src/less/main.less

```diff
@@ -4,6 +4,7 @@
 body {
     width: 100%;
     height: 100%;
+    overflow: hidden!important; // Important is to override base template
```

Comment:
This happens on the `drawer close` event of the app drawer, which happens automatically on the closing of the app drawer, or on window resize (triggers the drawer close).

At this point we would help suggestions on how to deal with the issue and how to remove the `!important`.